### PR TITLE
Add Chapter 3.C rules to "all the rules we know"

### PR DESCRIPTION
**Pull request by llewelld**
The usual rules dump; for 3C.

Preview of the changes:
https://github.com/llewelld/hut23-linear-algebra/blob/gh-action-result/pdf-files/reference/all-the-rules-we-know.pdf

### CODE REVIEW

File: reference/all-the-rules-we-know.tex [hunk 1]
```diff
@@ -746,4 +747,220 @@ \section*{Chapter 3.B}
...
+\begin{result}{3.35}[matrix of the sum of linear maps]
+Suppose $S, T \in \L(V, W)$. Then $\M(S + T) = \M(S) + \M(T)$.
+\end{result}
+
+\begin{result}{3.38}[the matrix of a scalar times a linear map]
+Suppose $\lambda \in \F$ and $T \in \L(V, W)$. Then $\M(\lambda T) = \lambda \M(T)$.
+\end{result}
+
+\begin{result}{3.40}[$\dim \F^{m, n} = m n$]
+Suppose $m$ and $n$ are positive integers. With addition and scalar multiplication defined as above, $\F^{m, n}$ is avector space of dimention $m n$.
```
Comment (triangle-man) [inline reference/all-the-rules-we-know.tex:870]: Typo: "dimention" should be "dimension"

**Review by triangle-man — COMMENTED** [review]
Looks good to me! I added comments where I spotted typos.

PS. for kernel and range the "modern" approach is `\DeclareMathOperator{\kernel}{null}`

File: reference/all-the-rules-we-know.tex [hunk 2]
```diff
@@ -746,4 +747,220 @@ \section*{Chapter 3.B}
 An inhomogeneous system of linear equations with more equations than variables has no solution for some choice of the constant terms.
 \end{result}
 
+\newpage
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\section*{Chapter 3.C}
+
+\begin{definition}{3.29}[matrix, $A_{j, k}$]
+Suppose $m$ and £n£ are nonnegative integers. An $m$-by-$n$ \defn{matrix} $A$ is a rectangular array of elements of $\F$ with $m$ rows and $n$ columns:
```
[the remark this answers is not in the document]
Comment (llewelld) [reply reference/all-the-rules-we-know.tex]: Thanks, nicely spotted; now fixed.

File: reference/all-the-rules-we-know.tex (hunk 1 above)
Comment (llewelld) [reply reference/all-the-rules-we-know.tex]: Likewise, thanks for flagging this. It's fixed now.

[the remark this answers is not in the document]
Comment (llewelld) [reply reference/all-the-rules-we-know.tex]: And again; thank you!

**Comment by llewelld** [comment]
Thanks for the review @triangle-man. I've fixed the typos and also updated the operator definitions. `\DeclareMathOperator` was new to me, so thanks for making me aware.

Also squashed and rebased.

**Comment by llewelld** [comment]
Rebased.

**Review by rwood-97 — APPROVED** [review]

**Comment by llewelld** [comment]
Thanks @rwood-97!

**Comment by llewelld** [comment]
Rebased.

**Pull request merged**